docs/Upgrade-Guide/README.md

```diff
@@ -4,6 +4,7 @@ Upgrading GlusterFS
 
 If you are using GlusterFS version 3.10.x or above, you can upgrade it to the following:
```

Comment:
please remove reference to 3.x here and in the other places of this file as well.